test_unstructured/partition/test_auto.py

```diff
@@ -1282,3 +1282,10 @@ def test_partition_image_with_bmp_with_auto(
     table = [el.metadata.text_as_html for el in elements if el.metadata.text_as_html]
     assert len(table) == 1
     assert "<table><thead><th>" in table[0]
+
+
+def test_auto_partition_eml_add_signature_to_metadata(filename="example-docs/eml/signed-doc.p7s"):
+    elements = partition(filename=filename)
```

Comment:
I know a lot of historical tests do this, but there's no reason to add filename as an argument to the test, at least that I've ever seen. I would just inline it like this:
```python
def test_auto_partition_eml_add_signature_to_metadata():
    elements = partition("example-docs/eml/signed-doc.p7s")
```

It's one less article of state to be kept in mind by the reader and reads very directly, like "partition this" :)

Reply:
Good call on that! Not sure when or why we started doing that.